src/nidcpower/metadata/functions.py

```diff
@@ -5518,7 +5179,6 @@ class driver recorded an interchangeability warning. Interchangeability
             },
             {
                 'direction': 'in',
-                'enum': None,
```

Comment:
It would have been much easier to update the extraction tools first to get all these little diffs out of the way, and then point to new driver version to get the content changes... Oh well.